openshift/pman-swift-publisher/put_data.py

```
@@ -35,28 +35,26 @@ def storeData(self, **kwargs):
             if k == 'out_dir':
                 outgoing_dir = v
 
-        # TODO: @ravig. The /tmp should be large enough to hold everything.
-        shutil.make_archive('/tmp/ziparchive', 'zip', outgoing_dir)
+        shutil.make_archive('/var/outgoingData', 'zip', outgoing_dir)
         try:
-            with open('/tmp/ziparchive.zip','rb') as f:
-                #TODO: @ravig - Change this so that this is scalable.
-                zippedFileContent = f.read()
-        finally:
-            os.remove('/tmp/ziparchive.zip')
+            f = open("/var/outgoingData.zip", "r")
+        except Exception as err:
+            print(err)
```

Should we be eating this exception?